pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/transformers/TeiPhysicalDescription.scala

```diff
@@ -0,0 +1,136 @@
+package weco.pipeline.transformer.tei.transformers
+
+import weco.pipeline.transformer.tei.NormaliseText
+
+import scala.xml.{Elem, Node, NodeSeq}
+
+object TeiPhysicalDescription {
+  def apply(xml: Elem): Option[String] =
+    apply(xml \\ "sourceDesc" \ "msDesc")
+  def apply(nodeSeq: NodeSeq): Option[String] = physicalDescription(nodeSeq)
+
+  /**
+    * The physical description can exist in msDesc for the wrapper work like in:
+    *<TEI xml:id="id" xmlns="http://www.tei-c.org/ns/1.0">
+    *   <teiHeader>
+    *     <fileDesc>
+    *       <sourceDesc>
+    *         <msDesc xml:lang="en" xml:id="MS_Arabic_1">
+    *           <physDesc>
+    *             <objectDesc>
+    *               <supportDesc>
+    *                 <extent>3 pages
+    *                   <dimensions unit="mm" type="leaf">
+    *                     <height>100mm</height>
+    *                     <width>300mm</width>
+    *                   </dimensions>
+    *                 </extent>
+    *               </supportDesc>
+    *             </objectDesc>
+    *           </physDesc>
+    *         </msDesc>
+    *       </sourceDesc>
+    *     </fileDesc>
+    *   </teiHeader>
+    * </TEI>
```

Comment:
Minor: I think you could delete all these lines so the file is more compact. These comments aren't syntactically correct; they're meant to help a human see that the code matches the data we're getting.